app/page.tsx

```diff
@@ -7,20 +9,53 @@ export const metadata: Metadata = {
 
 export default function Home() {
   return (
-    <main className="mx-5">
-      <div className="flex flex-col">
-        <h1 className="p-5 mt-5 text-4xl font-bold text-center">
-          The Decentralized Freelancing Platform
-        </h1>
-        <section className="mt-5">
-          <h2 className="text-2xl">Our Services</h2>
-          <ul className="pl-5 list-disc">
-            <li>Connect with global talents</li>
-            <li>Secure and trustless payment system</li>
-            <li>Full control over your contracts</li>
-          </ul>
-        </section>
+    <div>
+      <div className=" [background:linear-gradient(180deg,_rgba(214,_194,_188,_0.74),_rgba(0,_0,_0,_0.74))]">
+        <div className="grid grid-cols-1 sm:grid-cols-2">
+          <div className="relative justify-self-end order-first sm:row-span-2 sm:order-2">
+            <Image
+              alt="Bees"
+              src="/img/bees.png"
+              width="0"
+              height="0"
+              sizes="100vw"
+              className="w-full h-auto"
+            />
+          </div>
+          <div className="bg-comb flex flex-col content-around">
+            <div className="p-5 font-bold text-3xl">
+              <h1> Lorem ipsum dolor sit amet, consectetur adipiscing elit</h1>
+            </div>
+            <div className="p-5 font-bold text-xl">
+              <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
```

Comment:
We can use this content `More collaborative, more transparent and fairer than ever.` instead of the Lorem Ipsum.